prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 45 minutes
+Actual:   70 minutes
+"""
+
+words = input(str("Text: ")).split(' ')
+word_to_count = {}
+for word in words:
+    try:
+        word_to_count[word] += 1
+    except KeyError:
+        word_to_count[word] = 1
+words = list(word_to_count.keys())
```

Comment:
To avoid confusion, using a different name would be better as words is already defined.